desk.go

```diff
@@ -246,7 +246,8 @@ func Instance() Desktop {
 // The WindowManager passed in will be used to manage the screen it is loaded on.
 // An ApplicationProvider is used to lookup application icons from the operating system.
 func NewDesktop(app fyne.App, wm WindowManager, icons ApplicationProvider, screenProvider ScreenList) Desktop {
-	instance = &deskLayout{app: app, wm: wm, icons: icons, screens: screenProvider, settings: NewDeskSettings()}
+	instance = &deskLayout{app: app, wm: wm, icons: icons, screens: screenProvider}
```

Comment:
I'm not sure what this file has changed for, can you help?

Reply:
Because we want to make sure that an Instance exists before calling Settings now since we depend on the Instance in Settings.

Reply:
But the contents of NewDeskSettings() has not changed...

Reply:
NewDeskSettings calls settings.load() ... settings.load() need to be able to use Instance() to get the fallback default icons